cognite/neat/graph/loaders/_rdf2dms.py

```diff
@@ -0,0 +1,191 @@
+import json
+import warnings
+from collections import defaultdict
+from collections.abc import Iterable
+from pathlib import Path
+from typing import Any, cast
+
+import yaml
+from cognite.client import CogniteClient
+from cognite.client import data_modeling as dm
+from cognite.client.data_classes.data_modeling import ViewId
+from pydantic import HttpUrl, TypeAdapter, ValidationError, ValidationInfo, create_model, field_validator
+from pydantic.main import Model
+from rdflib.query import ResultRow
+from rdflib.term import URIRef
+
+from cognite.neat.graph.stores import NeatGraphStoreBase
+from cognite.neat.rules.issues import NeatValidationError
+from cognite.neat.rules.models import DMSRules
+from cognite.neat.rules.models.data_types import _DATA_TYPE_BY_DMS_TYPE
+
+from ._base import CDFLoader
+
+
+class DMSLoader(CDFLoader[dm.InstanceApply]):
```

Comment:
We should leverage in-memory pydantic models to both create assets/relationships and dms nodes and edges